src/test/test_projects/hello_world/pubspec.yaml

```diff
@@ -16,3 +16,7 @@ dependencies:
 
 dev_dependencies:
   test: '>=0.12.34'
+ # test
```

Comment:
This is left over from some tests that didn't clean up (perhaps they didn't complete correctly) and could be reverted.